app/common/lib/platformUtil.js

```diff
@@ -35,12 +35,12 @@ module.exports.getPathFromFileURI = (fileURI) => {
 
 module.exports.isDarwin = () => {
   return process.platform === 'darwin' ||
-    navigator.platform === 'MacIntel'
+    (navigator && navigator.platform === 'MacIntel')
 }
 
 module.exports.isWindows = () => {
   return process.platform === 'win32' ||
-    navigator.platform === 'Win32'
+    (navigator && navigator.platform === 'Win32')
 }
```

Comment:
These checks are put in place because if the first one fails (ex: calling isWindows on mac) from the browser process, it checks navigator (which was undefined).